docs/UMLdiagrams/MenuLogicSequenceDiagram.puml

```diff
@@ -0,0 +1,54 @@
+@startuml
+!include style.puml
+
+Actor User as user USER_COLOR
+
+Participant ":SubLogic(MenuLogic)" as subLogic SUB_LOGIC_COLOR
+Participant ":Parser" as parser PARSER_COLOR
+Participant ":Command" as command COMMAND_COLOR
+Participant ":Model" as model MODEL_COLOR
+
+
+user -> subLogic : "Create Menu"
```

Comment:
Should the "Create menu" arrow point to `MainLogic` instead since it's a `MainLogic` command? Then perhaps the `MainLogic` can then call the static method MenuLogic.modifyMenu() as seen in the code snippet below?

<img width="671" alt="image" src="https://github.com/AY2324S2-CS2113-F14-2/tp/assets/111105911/ca16cbc9-ea65-428e-a620-6d7a0d5b8793">